### ISSUE TITLE
Networking dialog on startup

### ISSUE THREAD

**Issue by asmaloney** [HaikuArchives/VMwareAddons#27]
Every time I start Haiku, VMWare throws up this dialog:

<img width="443" alt="Screen Shot 2021-06-23 at 10 45 41 AM" src="https://user-images.githubusercontent.com/391371/123312377-420b1d00-d4f6-11eb-9ff4-4b55f2936019.png">

Is there any way to silence this?

**Comment by korli** [comment]
Did you try the following?
```
"There's a checkbox under menu Fusion -> Preferences -> Network

"Require authentication to enable promiscuous mode"

**Comment by asmaloney** [comment]
Nope! I don't see that preference anywhere - there's no "Network" in my preferences. I'm using VMWare Fusion Version 11.5.7 (17130923) on macOS 10.14.6.

No other OS I've used in Fusion has required it - I don't like my OSes being promiscuous! (j/k - I have no idea what "promiscuous mode" means w.r.t. networking in this context.)

**Comment by volo-droid** [comment]
@asmaloney here it is:
|<img src="https://user-images.githubusercontent.com/14126089/134744586-a1ea6626-5798-465b-a18f-95d5d8d27724.png">|<img src="https://user-images.githubusercontent.com/14126089/134744452-49d3b8da-ff3d-4986-8945-822455c170ce.png">|
|---|---|

**Comment by asmaloney** [comment]
Thanks Volo.

I have VMWare Fusion Version 11.5.7 (17130923), and that section isn't there.
<img width="684" alt="Screen Shot 2021-09-24 at 20 56 37 PM" src="https://user-images.githubusercontent.com/391371/134752505-050b9fc8-26af-4869-b69a-d2936971acd0.png">

**Comment by diversys** [comment]
That tab is only available when you have the professional version of VMware Fusion.
It is not available on the free VMware Player.

See https://communities.vmware.com/t5/VMware-Fusion-Discussions/Where-did-the-Network-Settings-Disappear/td-p/2839562

**Comment by asmaloney** [comment]
I don't have the professional or the free - just the regular 'ol VMware Fusion that I paid for, but the pref doesn't exist there either.

So... back to the original question then :-) Given that this setting is not an option, is there another way to silence this dialog? Does Haiku really need this "promiscuous mode"?

**Comment by volo-droid** [comment]
For the same configuration, I don't have that popup regardless of the setting in the Network tab. What revision of Haiku are you running?

**Comment by asmaloney** [comment]
Huh - that's strange.

This is what I have:

<img width="692" alt="Screen Shot 2021-09-26 at 18 25 52 PM" src="https://user-images.githubusercontent.com/391371/134826205-5648db1d-5310-425e-9577-6a9735ce9c93.png">

**Comment by volo-droid** [comment]
Same here. What's the Network config (prefixed with `ethernet0.`) in your `.vmx` file? You could try to set the following parameters to the values I have: 
```
ethernet0.connectionType = "nat"
ethernet0.addressType = "generated"
ethernet0.linkStatePropagation.enable = "TRUE"
```
